src/components/lists/_macro.njk

```diff
@@ -9,7 +9,7 @@
       {% for item in (params.itemsList if params.itemsList is iterable else params.itemsList.items()) %}
           <li class="list__item {{ item.listclass }}" {% if item.current %}aria-current="true"{% endif %}>
               {% if item.url and item.current != true %}
-                  {% if item.index %}{{ loop.index }}. {% endif %}{% if item.prefix %}<span class="list__prefix" aria-hidden="true">{{ item.prefix }}.</span> {% endif %} <a href="{{ item.url }}" class="list__link {{ item.classes }}" {% if item.index %}data-qa="list-item-{{ loop.index }}"{% endif %} {% if item.target %} target="{{ item.target }}"{% endif %} {% if item.name %} name="{{ item.name }}"{% endif %}>
+                  <span class="list__prefix" aria-hidden="true">{% if item.index %}{{ loop.index }}. {% endif %}{% if item.prefix %}{{ item.prefix }}. {% endif %}</span><a href="{{ item.url }}" class="list__link {{ item.classes }}" {% if item.index %}data-qa="list-item-{{ loop.index }}"{% endif %} {% if item.target %} target="{{ item.target }}"{% endif %} {% if item.name %} name="{{ item.name }}"{% endif %}>
```

Comment:
We are outputting the prefix before checking for the prefix param...is this right?

Reply:
Yeah thats probably not right. I just wanted to make sure that if it is set using the `{{ loop.index }}` then it is also included in a span and has the correct classes. Ill rework this